야리무의 쇼핑몰/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>야리무 쇼핑몰</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/e126ff8299.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header>
+         <i class="fa-solid fa-heart-circle-plus"></i>
+         <div class="logo">
+            <h2> 야리무의 쇼핑몰 </h2>
+         </div>
+         <div class="burger">
+            <div class="line1"></div>
+            <div class="line2"></div>
+            <div class="line3"></div>
+         </div>
+         <div class="icon"></div>
+    
+    </header>
+
+    <nav>
+          <ul>
+           <p>판매목록</p>
+           <li><a href="#all">전체</a></li>
+           <li><a href="#base_img_wrapper">베이스</a></li>
+           <li><a href="#eye_img_wrapper">아이</a></li>
+           <li><a href="#color_img_wrapper">색조</a></li>
+          </ul>
+    </nav> 
+
+    <main>
+      
+         <ul>
+            <span>판매목록</span> 
+          
+          <section id="all">
+             <li class="title">전체</li>
+             <div class="bebe_wrapper">
+              <img src="./img/비비크림.jpg" width=100 height=100 />
+              <i class="fa-solid fa-heart"></i>
+              <p>닥터 자르트</p>
+              <p>24,000</p>
+             </div>
+             <div class="bebe_wrapper">
+              <img src="./img/쿠션.jpg" width=100 height=100 />
```

Comment:
이미지 태그에는 alt 속성값 넣어주기 !
이미지가 아직 로드되지 않은 경우나, 시각장애인 스크린리더에서 사용하려면 alt속성이 꼭 필요하답니다 !

```suggestion
              <img src="./img/쿠션.jpg" alt="쿠션이미지" width=100 height=100 />
```

Reply:
넵! alt 속성값 넣었습니다..! 감사합니다..!